user-ops-indexer/user-ops-indexer/migration/src/lib.rs

```diff
@@ -0,0 +1,81 @@
+pub use sea_orm_migration::prelude::*;
+use sea_orm_migration::sea_orm::{ConnectionTrait, Statement, TransactionTrait};
+mod m20220101_000001_blockscout_tables;
+mod m20220101_000002_initial_tables;
+mod m20231117_093738_add_indexes;
+
+pub struct Migrator;
+
+#[async_trait::async_trait]
+impl MigratorTrait for Migrator {
+    fn migrations() -> Vec<Box<dyn MigrationTrait>> {
+        vec![
+            Box::new(m20220101_000001_blockscout_tables::Migration),
```

Comment:
I didnt understand why you need blockscout tables inside your database.
Also, did you consider using `blockscout-db` crate (https://github.com/blockscout/blockscout-rs/tree/main/libs/blockscout-db) ?

Reply:
It'll be the same blockscout database as used in the core backend. CRUD queries include joins with blocks and logs tables, so the database has to be the same.

Full migration here is executed simply for local testing purposes, stilling figuring out how to make it work both locally and on real instances.

I think I tried using `blockscout-db` for this, but there were some compatibility issues between sea-orm versions. Will take one more look at it.

Reply:
This is now resolved:
1. Code uses `blockscout-db` model bindings for accessing blockscout tables
2. Tests use modified migration script with necessary database fixtures
3. Service migrations are tracked within `user_ops_indexer_migrations` table instead of the default `seaql_migrations` to avoid overlaps with other services in the future
4. Service migrations will fail if run on an empty database without required blockscout tables